src/frontend/src/utils/multiWebAuthnIdentity.ts

```diff
@@ -58,6 +59,36 @@ export class MultiWebAuthnIdentity extends SignIdentity {
     }
   }
 
+  /**
+   * This method exposes information about the underlying WebAuthn device. Only
+   * available after `sign()` has been called successfully.
+   * The 'authenticatorAttachment' is a field specified in the draft of the
+   * WebAuthn Level 3 spec (see
+   * https://w3c.github.io/webauthn/#dom-publickeycredential-authenticatorattachment)
+   * and already available on Chrome and Safari.
+   * Not yet implemented on Firefox.
+   *
+   * This method also exposes the credential id of the underlying device that was
+   * used so that the caller knows to which device the attachment applies.
+   */
+  public getAuthenticatorAttachment():
+    | undefined
```

Comment:
```suggestion
    undefined
```

Reply:
note: virtually everywhere else we have `<type> | undefined` (i.e. `undefined` in last position)